### ISSUE TITLE
Error: Execution blocked due to potentially dangerous command

### ISSUE THREAD

**Issue by conecta-web** [TiagoSilvaPereira/vemto-issues#17]
```
Error: Execution blocked due to potentially dangerous command
    at o.escapeCommand (app://./js/chunk-common.9dc99885.js:1:22684)
    at o.executeCommand (app://./js/chunk-common.9dc99885.js:1:21859)
    at app://./js/generator.e6e5039d.js:1:25175
    at new Promise (<anonymous>)
    at je.clone (app://./js/generator.e6e5039d.js:1:24915)
    at je.start (app://./js/generator.e6e5039d.js:1:24892)
    at pt.callGenerators (app://./js/generator.e6e5039d.js:1:39064)
    at app://./js/generator.e6e5039d.js:1:38758
    at Generator.next (<anonymous>)
    at r (app://./js/chunk-vendors.f2f83789.js:41:2263)
```

---

vemto 0.4.15
Electron 8.5.0
win32 10.0.17134
Locale: pt-BR

**Comment by TiagoSilvaPereira** [comment]
Hi @conecta-web could you please send me the folder name where you are saving the project? A think that it has some special character that is being blocked by Vemto for security when trying to execute a command

**Comment by TiagoSilvaPereira** [comment]
Fixed on 0.5.2
